cmd/ocm-backplane/console/console.go

```diff
@@ -249,6 +247,11 @@ func (o *consoleOptions) run(cmd *cobra.Command, argv []string) error {
 	if err != nil {
 		return err
 	}
+	// If this is a test scenario, we throw an error as opposed of keeping this function in a infinite loop
+	if len(argv) != 0 && argv[0] == "console_GO_test" {
```

Comment:
Maybe use a constant here instead of the raw `console_GO_test` flag?

Reply:
I'll make changes tomorrow